layouts/partials/sub-footer.html

```diff
@@ -7,7 +7,7 @@
             <li class="zerostatic"><a href="https://github.com/LeelaChessZero/">GitHub</a></li>
             <li class="zerostatic"><a href="https://discord.gg/pKujYxD">Discord</a></li>
             <li class="zerostatic"><a href="https://groups.google.com/forum/#!forum/lczero">Forum</a></li>
-            <li class="zerostatic"><a href="https://twitter.com/leelachesszero">Twitter</a></li>
+            <li class="zerostatic"><a href="https://x.com/leelachesszero">x.com</a></li>
```

Comment:
Maybe will make sense to replace them with icons, I don't really like the "x.com" text..
By the way, the "forum" is deactivated too.

Reply:
Do we already have an activityPub account, for example in Mastodon.social? We could add that here.